simapp/sim_test.go

```diff
@@ -139,6 +139,7 @@ func TestAppImportExport(t *testing.T) {
 	ctxA := app.NewContext(true, abci.Header{Height: app.LastBlockHeight()})
 	ctxB := newApp.NewContext(true, abci.Header{Height: app.LastBlockHeight()})
 	newApp.mm.InitGenesis(ctxB, app.Codec(), genesisState)
+	newApp.StoreConsensusParams(ctxB, consensusParams)
```

Comment:
Instead of having to make this method public (maybe it doesn't matter), perhaps instead we call `newApp.InitChain()` with the consensus params?

Reply:
I tried that on the beginning, but found several issues that was making the boilerplate too bid with the InitChainer. I can follow that but was too much iin my opinion.

I tried other ways too, using options, etc, but this was the less painful.